www/FingerPrint.js

```diff
@@ -57,21 +42,34 @@ FingerPrint.prototype.authenticate = function (successCallback, errorCallback,ob
 			errorCallback(error);
     };
     // if the device is android call the FingerprintAuth plugin
-    if(cordova.platformId === "android"){FingerprintAuth.encrypt(object, successCbFingerPrintAuth, errorCbFingerPrintAuth);}
+    if(cordova.platformId === "android"){Fingerprint.show(object, successCbFingerPrintAuth, errorCbFingerPrintAuth);}
     // if the device is ios call the touchid plugin 
     if(cordova.platformId === "ios")touchid.authenticate(successCallback, errorCallback, object.dialogMessage);
 };
 
 FingerPrint.prototype.checkBiometry = function(successCallback, errorCallback) {
-    // success callback function for android 
-    var successCbFingerPrintAuth = function(result) {successCallback(1)};
-    // error callback function for android 
-    var errorCbFingerPrintAuth = function(error) {
-			errorCallback(-1);
-    };
   
-    // if the device is android call the FingerprintAuth plugin
-    if(cordova.platformId === "android"){FingerprintAuth.isAvailable(successCbFingerPrintAuth, errorCbFingerPrintAuth);}
+    function isAvailableSuccess(result) {
+      var biometricType = 3;      
+      if (result === "biometric") {
+        biometricType = 2;
+        successCallback(biometricType);
+      }
+      else if (result === "finger") {
+        biometricType = 1;
+        successCallback(biometricType);
+      }
+      else {
+        successCallback(biometricType);
+      }
```

Comment:
"Finger" is touchID?

Its a bit confusing since if it is touchID then its also a "biometric".

Reply:
Agreed.
This is, most likely, an unfortunate nomination.

First, there was only fingerprint sensors, and then, other biometric sensors came. 
What it means when a device has biometricType **biometric**? 
It means it has both fingerprint sensor and face sensor? Something like that?

Reply:
This is due to a slightly different operation on the part of android compared to iOS. The in Android plugin returns just "biometric" or "finger". This means that if the return is biometric, face our finger can be used, but it will depend of device configuration.